### PR TITLE
Fix #806

### LINKED ISSUE #806: PyAlbany compilation error
It looks like PyAlbany failed to compile last night due to a missing include:

```
/nightlyCDash/repos/Albany/PyAlbany/src/PyAlbany_Stokhos.hpp:10:10: fatal error: Stokhos_KL_ExponentialRandomField.hpp: No such file or directory
```
https://sems-cdash-son.sandia.gov/cdash/viewBuildError.php?buildid=33107

My guess is the file included either got removed or renamed in Trilinos.  Do we really need the include?  Would be great if you could have a look @kliegeois .

### PR DESCRIPTION
**Pull request by kliegeois**
This should fix #806

### CODE REVIEW

File: PyAlbany/src/PyAlbany_Stokhos.cpp [hunk 1]
```diff
@@ -3,6 +3,7 @@
 using namespace PyAlbany;
 
 KLExpansion::KLExpansion(int _ndim) : ndim(_ndim) {
+#ifdef ALBANY_STOKHOS
```
Comment (bartgol) [inline PyAlbany/src/PyAlbany_Stokhos.cpp:6]: All these ifdefs are pointless, no?

**Review by bartgol — APPROVED** [review]
Looks good. I have minor comment, but nothing serious.

**Review by mperego — COMMENTED** [review]
Thanks @kliegeois. 
If  Stokhos is not enabled, then the class KLExpansion would not work as expected. Isn't it better to avoid compiling the files Albany_Stokhos.cpp and Albany_Sthokos.hpp?

Is this code used at all? If it's used in a test we should disable the test as well when ALBANY_STOKHOS is false.

**Comment by kliegeois** [comment]
There is no tests that use those. The main issue and the raison why I did it with so many ifdef is swig. I am 90% sure that only one per file would not compile PyAlbany correctly; I am not 100% sure though.

**Comment by kliegeois** [comment]
I can try to reduce the number of ifdef though.

**Comment by mperego** [comment]
Thanks. We really should have a test for it. No need to do add the test in this PR, though.

**Comment by ikalash** [comment]
I merged this in.  Will check tomorrow if the failed nightly is fixed.

**Comment by kliegeois** [comment]
It should fix the issue but it is not yet working perfectly. It will build and run the tests correctly.

**Comment by mperego** [comment]
@ikalash we were still discussing about this PR. That's why Kim added the WIP label. Not a big deal, but I think it's in general better not to merge PR's by others without checking first.

**Comment by ikalash** [comment]
Oh shoot...  I missed the WIP label.  Sorry about this - somehow the commenting made me erroneously believe it was ready to be merged.  I can revert the merge if you'd like.

**Comment by bartgol** [comment]
I think we can keep it in now. If Kim has additional commits, he can submit a separate PR, I guess.

**Comment by mperego** [comment]
No worries. No need to revert, thanks.

**Pull request merged**